readme.txt

```diff
@@ -1,11 +1,11 @@
 # Checkfront Online Booking System
 
 Contributors: checkfront
-Stable tag: 3.7
+Stable tag: 3.8
 Tags: Booking, Booking System, Reservation, Reservation System, Online Booking, Booking Engine, Tours, Tour Operator, Booking Plugin, Reservation Plugin, Booking Software, Reservation Payment System, Activity Booking, Rental Booking, Reservation Payments, Tour Booking, Passbook, Availability, Payments, Bookings
 Requires at least: 2.0
 License: GPLv2 or later
-Tested up to: 6.3.1
+Tested up to: 6.4.0
```

Comment:
🌵 IIRC if you omit the `.0`, it still works and becomes stale less frequently. Might need to do some quick research to confirm though.

Reply:
I'll take a look at the Wordpress docs

Reply:
Yep, dropping the last minor version should cause it to auto update, I'll change that now